gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Field.java

```diff
@@ -72,6 +79,8 @@ public boolean equals(Object o) {
     return name().equals(other.name())
         && originalName().equals(other.originalName())
         && type().equals(other.type())
+        && isRequired() == other.isRequired()
+        && fieldInfoFormat() == other.fieldInfoFormat()
```

Comment:
We need to add these two to the hashcode method below as well if we continue to use these manually written hashcode/equals methods

Reply:
Ah I see - I'm not clear how the integers are chosen to create the hashing (if we do decide to use these manual methods). We can discuss offline.

Reply:
Added the fields to the `hashCode()` method below as discussed yesterday.